content/docs/reference/service-urls.md

```diff
@@ -0,0 +1,287 @@
+---
+title: Service URLs
```

Comment:
Should this be "Service URL Settings" to match the page heading below?

Reply:
It only affects how the title tag is rendered, but yes, these should be consistent. Updated.